<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightWindow</class>
 <widget class="QMainWindow" name="FightWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="backgroundView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dialogLabel">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>446</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Dialog</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>440</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN!!</string>
    </property>
    <property name="skill" stdset="0">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="skillButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
    <property name="skill" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="skillButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
    <property name="skill" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="skillButton_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
    <property name="skill" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="skillButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
    <property name="skill" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="playerHealthBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>55</number>
    </property>
    <property name="format">
     <string>%v/%m</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="enemyHealthBar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v/%m</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="playerMagicBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v/%m</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="enemyMagicBar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="format">
     <string>%v/%m</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Player HP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Player MP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enemy MP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enemy HP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit to Board</string>
    </property>
    <property name="skill" stdset="0">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="gameOverBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Game Over!</string>
    </property>
    <widget class="QPushButton" name="gameOverButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Return to menu</string>
     </property>
     <property name="skill" stdset="0">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="gameOverDesc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
